tests/test_indexer.py

```diff
@@ -327,6 +347,65 @@ def get_search_results(cls, query, expected_hit_count):
                 time.sleep(0.5)
 
 
+class BundleBuilder:
```

Comment:
IMO, we should consider making this a general purpose class.  bundle.py would probably benefit from it.